dist/style.css

```diff
@@ -0,0 +1,149 @@
+/* CSS HEX */
+:root {
+  --alabaster: #d8e2dcff;
+  --icy-white: #f7f7f7;
+  --yellow-green-crayola: #cef7a0ff;
+  --cyan-blue: #b2dce5;
+}
+
+* {
+  border: 0px;
+  margin: 0px;
+  box-sizing: border-box;
+  margin-block-start: 0px;
+  margin-block-end: 0px;
+  font-family: "Fredoka One", cursive;
+}
+
+body {
+  background-image: url("./assets/images/background-img.png");
+  height: 100%;
+  width: 100%;
+}
+
+.header-banner {
+  display: flex;
+  justify-content: center;
+  padding: 3rem;
+  background-color: var(--yellow-green-crayola);
+  width: 100%;
+}
+
+.header {
+  font-family: "Fredoka One", cursive;
+}
+
+.main {
+  margin: 2rem 1rem;
+}
+
+/* card properties */
+.card {
+  position: relative;
+  top: 5vh;
+  box-shadow: 0 10px 20px rgba(0, 0, 0, 0.19), 0 6px 6px rgba(0, 0, 0, 0.23);
+  border-radius: 0.75em;
+  padding: 0;
+  transition: all 0.2s ease-in-out;
+  width: 20rem;
+  margin: 0rem 2rem 2rem 2rem;
+}
+.card:nth-child(even) {
```

Comment:
Very picky but please be consistent with spacing and make sure there is a blank line between each block